addon/components/inputs/checkbox-array.js

```diff
@@ -5,6 +5,14 @@ import layout from 'ember-frost-bunsen/templates/components/frost-bunsen-input-c
 import Ember from 'ember'
 const {get} = Ember
 
+export const helpers = {
+  validateChoices (choices, meta) {
+    if (!Ember.isEmpty(choices) && !Ember.isEmpty(meta)) {
```

Comment:
Change line 6 to be `const {get, isEmpty} = Ember` and just use `isEmpty` in this line.